libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef.dat

```diff
@@ -266,6 +266,7 @@ define HAL_BATT_CURR_PIN 3
 define HAL_BATT_VOLT_SCALE 10.1
 define HAL_BATT_CURR_SCALE 17.0
 define HAL_GPIO_PWM_VOLT_PIN 3
+define HAVE_DUAL_CDC 1
```

Comment:
as mentioned previously, this should start with HAL_HAVE_ to keep it in HAL namespace.
It also should not need to be added in hwdef.dat, it should be automatic when the parser sees that OTG2 is added to the uart list.